cmake/run_main.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Run program with CMakeLists.cpp
+
+if [ -d ./build ]
+then
+	rm -r build
+fi
+
+mkdir build
+
+cd build
+cmake ..
```

Comment:
Your script's name and what it does differ.  Script which is written to run the project must not build it.
There are 2 options:
1. Your are writing a script `build.sh` which builds the project. and this script simply runs it.
2. You are not writing a `build.sh` script as It is too simple to build it. So you can write build commands in the readme

Reply:
+